src/Forwarder.h

```diff
@@ -15,8 +15,8 @@ namespace Forwarder {
 
 class MappingAddException : public std::runtime_error {
 public:
-  explicit MappingAddException(std::string what)
-      : std::runtime_error(std::move(what)) {}
+  explicit MappingAddException(std::string const &What)
```

Comment:
I think this is a regression, we talked about it at the ECP meeting in Lund. https://www.bfilipek.com/2018/08/init-string-member.html